### PR TITLE
Update for new version

### PR DESCRIPTION
**Pull request by zdmullen**
Signed-off-by: Zachariah Mullen <zachariah.mullen@broadcom.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 This file includes the major changes to the Eclipse Che4z Basic and Premium stacks.
 
+## [2.1] 2020-08-21
+
+- Zowe Explorer updated to version 1.8.0
+- Explorer for Endevor updated to version 0.10.0. Users can now utilize their CA Endevor Plug-in for Zowe CLI profiles in Explorer for Endevor. ([Changelog](https://github.com/eclipse/che-che4z-explorer-for-endevor/blob/master/CHANGELOG.md)).
+- COBOL Language Support updated to version 0.12.0 ([Changelog](https://github.com/eclipse/che-che4z-lsp-for-cobol/blob/development/CHANGELOG.md)).
```
Comment (zimlu02) [inline CHANGELOG.md:7]: 0.13.0

**Comment by zeibura** [comment]
Is this going to have Debugger 1.2 in it as well?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,12 @@
...
+
```
Comment (zimlu02) [inline CHANGELOG.md:8]: could you also attach Debugger for Mainframe v1.2.0?

Comment (zdmullen) [reply CHANGELOG.md:9]: done

File: CHANGELOG.md (hunk 1 above)
Comment (zdmullen) [reply CHANGELOG.md]: done

**Comment by zeibura** [comment]
Debugger 1.2 required some changes to the readme. I have made them, please have a look.

**Comment by zimlu02** [comment]
> Is this going to have Debugger 1.2 in it as well?

let me put it there

**Review by zimlu02 — APPROVED** [review]

**Review by zeibura — APPROVED** [review]

**Pull request merged**